mobile-version.html

```diff
@@ -0,0 +1,124 @@
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">javascript</a></li>
+                    </ul>
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+            <div class="job-3">
+                <img src="/images/Snapshoot-Portfolio-3.jpg" alt="job-3 image">
+                <div class="content">
+                    <h3>Tonic</h3>
+                    <ul>
+                        <li>Canopy</li>
+                        <li>Back End DEv</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul>
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">javascript</a></li>
+                    </ul>
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+            <div class="job-4">
+                <img src="/images/Snapshoot-Portfolio-2.jpg" alt="job-4 image">
+                <div class="content">
+                    <h3>Multi-Post Stories</h3>
+                    <ul>
+                        <li>Canopy</li>
+                        <li>Back End DEv</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul>
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">javascript</a></li>
+                    </ul>
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+        </section>
+        <section class="About">
+            <h2>About <br> Myself</h2>
+            <p class ="description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <a href="#" class="connect">LET'S CONNECT</a>
+                <div class="contacts">
+                    <ul>
+                        <li><a href="#" class ="link"><i class="fa fa-twitter"></i></a>
+                        <li><a href="#" class ="link"><i class="fa fa-linkedin-square" ></i></a></li>
+                        <li><a href="#" class ="link"><i class="fa fa-github"></i></a></li>
+                        <li><a href="#" class ="link"><i class= "fa fa-hand-peace-o"></i></a></li>
+                        <li><a href="" class="link"><i class="material-icons">feedback</i></a></li>
+                    </ul>
+                </div>
+                <button type="button">Get Resume</button>
```

Comment:
- [ ] I think it would be better if you stick with the Figma template as here so update the Button to have the same `placeHolder` and the same padding💪🏻💪🏻